bus/bus.go

```diff
@@ -98,6 +108,28 @@ type bus struct {
 	cs  ContractStore
 	css ContractSetStore
 	os  ObjectStore
+	ss  SettingStore
+}
+
+func (b *bus) loadDefaultSettings() (err error) {
```

Comment:
Updatable default settings in a db seem a bit odd to me. I'd rather have the user be explicit when they call the endpoints and notify them in case they failed to provide certain settings such as redundancy.

Reply:
Currently we don't allow the user to pass in the redundancy, that's why I figured the `bus` would need some default redundancy settings. I think even if we allow to pass them we could use defaults so the redundancy parameters can be optional.

Reply:
resolving this, think it's moot at this point